source/apiVolontaria/volunteer/tests/tests_view_ParticipationsId.py

```diff
@@ -51,6 +58,14 @@ def setUp(self):
             name="my cell",
             address=self.address,
         )
+        self.cell_with_manager = Cell.objects.create(
+            name="my cell with manager",
+            address=self.address,
+        )
+        self.cell_with_manager.save()
+        self.cell_with_manager.managers = [self.user_cell_manager, ]
+        self.cell_with_manager.save()
```

Comment:
You don't have to call `.save()` twice here. You can keep the second only.

Reply:
Done ;)